src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TracePoint.kt

```diff
@@ -179,10 +179,10 @@ internal class ReadTracePoint(
 }
 
 internal class WriteTracePoint(
-    private val ownerRepresentation: String?,
+    var ownerRepresentation: String?,
```

Comment:
Let' try to keep it `private` and `val`, and apply all necessary transformations upon initialization.

Reply:
see https://github.com/JetBrains/lincheck/pull/683#issuecomment-2983300439